README.md

```diff
@@ -220,8 +269,4 @@ There's more than one way to give back, from examples to blogs to code updates.
 ## License
 [Snap](http://github.com/intelsdi-x/snap), along with this plugin, is an Open Source software released under the Apache 2.0 [License](LICENSE).
 
-## Acknowledgements
-* Authors: [Samantha Alt](https://github.com/saalt),
-           [Joel Cooklin](https://github.com/jcooklin)
```

Comment:
Why this section is removed? FYI, it still available in contents in the beginning of README (see [here](https://github.com/intelsdi-x/snap-plugin-collector-pysmart/pull/13/files#diff-04c6e90faac2675aa89e2176d2eec7d8R17))